package-testing/python-sdk-relay/build-and-run.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
minor: fail if anything goes wrong?
```suggestion
#!/usr/bin/env bash
set -euxo pipefail
```

Reply:
💪